extensions/applicationinsights-analytics-js/src/JavaScriptSDK/AnalyticsPlugin.ts

```diff
@@ -284,17 +284,29 @@ export class AnalyticsPlugin extends BaseTelemetryPlugin implements IAppInsights
                 if (doc) {
                     pageView.refUri = pageView.refUri === undefined ? doc.referrer : pageView.refUri;
                 }
-        
+
+                let perf = getPerformance();
+                 // Access the performance timing object
+                const navigationEntries = (perf && perf.getEntriesByType && perf.getEntriesByType("navigation")); 
+                if (navigationEntries) {
+                    // Get the value of loadEventStart
+                    const navigationEntry = navigationEntries[0] as PerformanceNavigationTiming;;
+                    const loadEventStart = navigationEntry.loadEventStart;
+                    pageView.startTime =  new Date(perf.timeOrigin + loadEventStart);
+                } else {
+                    // calculate the start time manually
+                    pageView.startTime = new Date(new Date().getTime() - pageView.properties.duration);
```

Comment:
```suggestion
                    pageView.startTime = new Date(new Date().getTime() - properties.duration);
```

Reply:
Looking at all of the usages of the sendPageInternal and trackPageView, while generally (when there is a `properties`) the `properties.duration` and `pageView.properties.duration` are the same, it is not always the case.

But reviewing the code I did notice that it is very possible and likely that both `pageView.properties` and `properties` may be undefined. So we should protect against that and also use a fallback, keeping the current loging of the pageView.properties.duration and the primary.

Reply:
((properties || paveView.prperties || {}.duration || 0)

Reply:
The tests are failing because of dependencies `rush update --recheck --purge --full` should fix